tests/integration/test_pip_packages.py

```diff
@@ -0,0 +1,116 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from os import path, listdir, walk
+from requests import get
+import tarfile
+
+from utils import Client
+
+
+def test_pip_package_without_deps(test_env, tmpdir):
+    """
+    Validate data in the pip package request without dependencies.
```

Comment:
Should we add params to the docstring?

Reply:
I'm fine if you want to skip them. I only mentioned since the other assertion functions in the file has description of what is being passed to them as params.

Reply:
I think we don't need to add comments for these params, it's not in other tests too.

Reply:
@nirzari we have normal parameters that we fill by ourselves in assertion functions. In opposite to it in test functions, we have special parameters - pytest test functions. `test_env` is not a parameter but a function. If you use pycharm as your local IDE you can click on it and check.